source/2019-04-12-the-ember-times-issue-94.md

```diff
@@ -86,13 +86,16 @@ responsive: true
   
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [User Impersonation in SaaS with Ember.js and Ruby on Rails 🐹](https://codeandtechno.com/posts/user-impersonation-ember-simple-auth-doorkeeper/)
+
+When developing applications it is important to be able to log in as one of your users and see exactly what they see. This can help debugging issues and aid your support team in assisting users. 
+
+[@matixmatix](https://github.com/matixmatix) has created a awesome tutorial on how this can be done using Ember.js ([Ember Simple Auth](https://ember-simple-auth.com/) with Ruby on Rails (with [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) gem FOR 0Auth2). 
+
+Achieving this is fairly simple process. Once the client is exchanging username and password for an access token we then call this 0Auth 2.0 Password grant flow. Then to impersonate a user we then create a new token by bypassing the part where the password is used. Then by asking DoorKeeper to issue a new token for the provided user directly. Now we have a token we then create a new authenticator by extending the existing 0Auth2 authorizer, so we can pass the tokenData directly to our Ember app. 
```

Comment:
Explaining OAuth2 isn't easy (there's always _that diagram_ that confuses me), so good job summarizing the important steps!

For clarity, I might suggest the following:

"Achieving this is a fairly simple process. Normally, a client exchanges their username and password for an access token. To impersonate that user, we would need to create the token without the user's password. We can ask DoorKeeper to issue a new token for the provided user directly. Lastly, we update our existing OAuth2 authorizer so that we can pass the token (instead of password) to log in as that user."